terms/abs/abs.md

```diff
@@ -0,0 +1,51 @@
+## Syntax
+```cs
+Math.Abs(value)
+```
+Math.Abs() is a static method of the Math class. It takes a single parameter of type int, long, float, double, or decimal. It returns the absolute value of the number.
+
+## Example:
+Find the absolute value of the number -5.
+```cs
+int x = -5;
+int y = Math.Abs(x);
+Console.WriteLine(y);
+```
+
+Codebyte Exmaple:
+```codebyte/csharp  
```

Comment:
There should be a sentence introducing the codebyte describing what it does. It should also be different than the example given above.